www/src/pages/foundations/colors.tsx

```diff
@@ -57,30 +55,26 @@ export interface ISwatch {
   isUnused?: boolean,
 }
 
+const styles = getComputedStyle(document.body);
```

Comment:
[curious] When running this locally, when switching between the "Open edX" and "edX" themes, the HEX values don't appear to update, though the colors do. Is this potentially due to `styles` getting called outside of the React component?

Reply:
Perhaps this problem is no longer relevant. I checked and this is what I got. Please take a look at the screen recording.
Uploading Screen Recording 2023-09-08 at 1.14.30 PM.mov…